src/header/header.scss

```diff
@@ -190,181 +136,191 @@
     }
   }
 }
 
-@mixin iui-header-button-focus {
-  &:focus {
-    box-shadow: none;
-    outline-offset: -1px;
-
-    @include themed {
-      outline: 1px solid t(iui-color-foreground-primary);
-    }
-  }
-
-  &:focus:not(:focus-visible) {
-    outline: 0;
-  }
-}
-
-@mixin iui-header-button {
+@mixin iui-header-breadcrumbs-button {
+  @include iui-button;
+  @include iui-button-borderless;
   @include iui-button-size(large, borderless);
-
+  @include iui-header-button-reset;
   gap: 0;
   max-width: 25vw;
-  height: 100%;
-  border-radius: 0;
   overflow: hidden;
   padding-right: $iui-s;
 
-  @include iui-header-button-focus;
+  &-name,
+  &-description {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    transition: color $iui-speed-fast ease-out;
+  }
+
+  &-description {
+    height: $iui-baseline * 2;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: height $iui-speed-fast ease-out;
+    }
+    @include themed {
+      opacity: t(iui-opacity-3);
+    }
+  }
 
   > .iui-button-label {
     text-align: left;
     font-size: $iui-font-size;
     overflow: hidden;
 
-    > div {
-      overflow: hidden;
-      text-overflow: ellipsis;
-    }
-
-    .iui-description {
-      height: $iui-baseline * 2;
-      @media (prefers-reduced-motion: no-preference) {
-        transition: all $iui-speed-fast ease;
-      }
-      @include themed {
-        opacity: t(iui-opacity-3);
-      }
+    @media (max-width: 768px) {
+      margin-left: 0;
     }
   }
 
   > .iui-button-icon {
     flex-shrink: 0;
+    box-sizing: content-box;
   }
 
   > * + * {
     margin-left: $iui-sm;
   }
 
   .iui-header-button-icon {
+    display: inline-flex;
     padding: $iui-xs;
+    box-sizing: content-box;
     @include iui-header-button-icon($iui-icons-default, $iui-border-radius);
     @include themed {
       background-color: t(iui-color-background-3);
       fill: t(iui-icons-color);
     }
+
+    @media (max-width: 768px) {
+      display: none;
+    }
+  }
+
+  img.iui-header-button-icon {
+    padding: 0;
+    width: auto;
+    height: $iui-icons-default + ($iui-xs * 2);
   }
 
   &.iui-active {
     @include iui-header-button-active;
   }
 
-  &:disabled {
+  &.iui-disabled,
+  &:disabled,
+  &[aria-disabled='true'] {
+    cursor: not-allowed;
+
     .iui-header-button-icon {
       filter: grayscale(1) opacity(0.7);
     }
   }
 
   @at-root {
-    .iui-page-header.iui-slim {
-      > .iui-left {
-        .iui-button {
-          .iui-header-button-icon.iui-header-button-icon {
-            width: $iui-icons-small;
-            height: $iui-icons-small;
-            padding: $iui-xxs;
-          }
+    .iui-slim {
+      .iui-header-breadcrumbs-button {
+        .iui-header-button-icon {
+          width: $iui-icons-small;
+          height: $iui-icons-small;
+          padding: $iui-xxs;
+          margin: 0 $iui-xs;
+        }
+
+        img.iui-header-button-icon {
+          width: $iui-icons-small + ($iui-xxs * 2);
```

Comment:
This is inconsistent.. On normal size you set width as `auto`, so maybe it should have width defined in exact number in both cases?